fedimint-server/src/config/mod.rs

```diff
@@ -208,7 +208,7 @@ impl ServerConfig {
     pub fn supported_api_versions() -> SupportedCoreApiVersions {
         SupportedCoreApiVersions {
             core_consensus: CORE_CONSENSUS_VERSION,
-            api: MultiApiVersion::try_from_iter([ApiVersion { major: 0, minor: 1 }])
+            api: MultiApiVersion::try_from_iter([ApiVersion { major: 0, minor: 2 }])
```

Comment:
Do we also need to bump the client core api version?

https://github.com/fedimint/fedimint/blob/427f09b/fedimint-client/src/lib.rs#L659

Reply:
I don't think so, we'd do that once we drop support for 0.0 afaik. @dpc